### DISCUSSION TITLE
Running prisma migrate deploy using Docker

### DISCUSSION THREAD

**Discussion by jeepers3327** [prisma/prisma#8355] [Q&A] [answered]
Hi, 

I'm on my way on dockerizing my app for deployment wtih docker-compose and everything works fine until I notice that I haven't run `prisma migrate deploy` 

Here's my Dockerfile
```Dockerfile
FROM node:14.17-alpine3.13 as development

WORKDIR /app

COPY package.json package.json
COPY yarn.lock yarn.lock
COPY prisma ./prisma/
COPY .env.production ./.env

RUN yarn install
RUN npx prisma generate

COPY . .
RUN yarn run build


FROM node:14.17-alpine3.13

WORKDIR /app

COPY --from=development /app/node_modules ./node_modules
COPY --from=development /app/package.json ./
COPY --from=development /app/yarn.lock ./
COPY --from=development /app/dist ./dist
COPY --from=development /app/prisma ./prisma


CMD [ "yarn", "run", "start:prod" ]
```

Then after running `docker-compose up`, I execute `docker exec -it backend bash` and run `npx prisma migrate deploy`.

Do you have any resources/repo that runs the `npx prisma migrate deploy` after `docker-compose up`

**Comment by ryands17** [comment, marked as answer]
@jeepers3327 👋 

You can run `prisma migrate deploy` in your `start:prod"` command itself. This will make sure that it will be run whenever you run `docker compose up`.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
